shell/platform/tizen/flutter_tizen_engine.h

```diff
@@ -86,6 +73,10 @@ class FlutterTizenEngine : public TizenRenderer::Delegate {
   // Returns the currently configured Plugin Registrar.
   FlutterDesktopPluginRegistrarRef GetPluginRegistrar();
 
+  FlutterTizenTextureRegistrar* texture_registrar() {
```

Comment:
I suggest as below
```suggestion
  FlutterTizenTextureRegistrar* GetTextureRegistrar() {
```

Reply:
Good suggestion, I have rename this function.

Reply:
I liked it because it had a similar name to the other implementation..